### PR TITLE
doc: Document tenant management

### LINKED ISSUE #347: Document tenant management
#### Summary

Document basic tenant management.

#### Why do we need this ?

For TTSE multi-tenant customers to do basic tenant management that is not provided by the tenant billing server (or if they're not using TBS).

#### What is already there? What do you see now?

Installation instructions for TTSE CLI (#345)

#### What is missing? What do you want to see?

Basic tasks, like:
1. Refer to installing the TTSE CLI and how authentication works for this (i.e. not OAuth 2.0 but via tenant admin key)
2. CRUD tenants
3. Configure DevAddr prefixes for a tenant

#### How do you propose to document this?

1. New section under Reference, Enterprise distribution only
2. Extend `cli-only` shortcode with option to include that it needs the TTSE CLI. You might want to do this via a parameter to the shortcode, just like we have one for instructing an upgrade
3. Document that tenant management does not work with user authentication (OAuth 2.0) but with the `--tenant-admin-key` (or `tenant-admin-key: XXX` in `.ttn-lw-cli.yml`)
4. Document basic actions like `tti-lw-cli tenant list`, get, update, delete
5. Document setting a DevAddr prefix: like `tti-lw-cli tenant update xxx --configuration.default-cluster.ns.dev-addr-prefixes 27123400/24`

@nejraselimovic please coordinate with @adriansmares

#### Can you do this yourself and submit a Pull Request?

Can review

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Closes #347 

#### Changes
Documenting basics for tenant management, like authentication and basic operations.

#### Checklist
- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by johanstokking** [comment]
> * Can you explain why configuring a DevAddr prefix for a tenant is important enough to be highlighted? I see how to achieve it, but I'm not sure I fully understand what it means, so please give me an input.

It's for Packet Broker. Tenants are using the same NetID, but they are distinguished by the DevAddr block that they use. Network Server issues a DevAddr in the range of the tenant. Packet Broker then knows to which tenant the message belongs, based on the DevAddr.

File: doc/content/reference/tenant-management/_index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+This means all tenant management CLI commands will need to have the `--tenant-admin-key` flag appended, followed by a tenant admin key that you configured in the Identity Server. See the section below for examples.
+
+## Basic Operations
+
+### Create a Tenant
+
+To create a new tenant with the tenant ID `tenant1`:
+
+```
+$ tti-lw-cli tenant create tenant1 --tenant-admin-key 14596c0bdb9b11f6a97273e7c09167531571efa5898274e48660ee8d08f62b67
```
Comment (johanstokking) [inline doc/content/reference/tenant-management/_index.md:28]: Probably useful to declare a local variable;

```bash
$ tenantadminkey=14596c0bdb9b11f6a97273e7c09167531571efa5898274e48660ee8d08f62b67
$ tti-lw-cli tenant create tenant1 --tenant-admin-key $tenantadminkey
```

@adriansmares we don't respect an env key for this right?

Comment (adriansmares) [reply doc/content/reference/tenant-management/_index.md:28]: Correct, this is just an extra parameter added to the tenant commands - not a config parameter (when used like here).

Comment (KrishnaIyer) [reply doc/content/reference/tenant-management/_index.md:28]: Indeed it doesn't respect the `TTN_LW_TENANT_ADMIN_KEY` env.

Another tip on macOS is to copy the tenant admin key (to the clipboard) and use `pbpaste`.
```
tenantadminkey=$(pbpaste)
$ tti-lw-cli tenant create tenant1 --tenant-admin-key $tenantadminkey
```

**Review by adriansmares — APPROVED** [review]

**Pull request merged**